core/arch/riscv/tee/arch_ecall_rv64.S

```diff
@@ -0,0 +1,66 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright  2022  Beijing ESWIN Computing Technology Co., Ltd.
+ */
+#include <riscv.h>
+#include <asm.S>
+#include <generated/asm-defines.h>
+#include <tee_api_defines.h>
+
+/*
+ * uint32_t tee_ecall_do_call(struct thread_ctx_regs *regs, syscall_t func);
+ *
+ * Called from user_ta_handle_svc()
+ */
+FUNC tee_ecall_do_call , :
+UNWIND(	.cantunwind)
+	mv	t0, a0
+	mv	t1, a1
+	addi	sp, sp, -8
```

Comment:
Should sp be 16-byte aligned?
(See latest RISC-V unprivileged ISA specification section 18.5 - Integer Computational Instructions)